easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-11.2.0-i4.eb

```diff
@@ -0,0 +1,57 @@
+# this recipe is configured for use with Gaussian G16, revision A03 or later
+
+easyblock = 'MakeCp'
+
+name = 'NBO'
+version = '7.0.10'
+versionsuffix = '-i4'
+
+homepage = 'http://nbo.chem.wisc.edu/'
+description = """ The Natural Bond Orbital (NBO) program is a discovery tool
+for chemical insights from complex wavefunctions. """
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+download_instructions = """
+NBO7 is proprietary software, order here: https://charge.wisc.edu/chemistry/order_nbo7.aspx
+The dowloaded tarball does not contain the bug-fix version so rename to nbo7.0.10-src.tar.gz
+"""
+sources = ['nbo%(version)s-src.tar.gz']
+patches = [
+    'NBO-%(version)s_make.patch',
+]
+checksums = [
+    {'nbo7.0.10-src.tar.gz': 'a60d1f4d4592ec386b5378841d9fba75918c48b50de08d8de0a50db42bc96da1'},
+    {'NBO-7.0.10_make.patch': 'c37f5a9980c56198f9610611b0fbda8ad65c961ec19b163d602dc3330fced145'},
+]
+
+dependencies = [
+    ('FlexiBLAS', '3.0.4'),
+]
+
+local_intlength = 'i4'  # changed to i4
+
+buildopts = 'FC=$FC CC=$CC NBODIR="%(builddir)s/nbo7" STATIC="false" PROFILE="true" '
+buildopts += 'BLASLIB="$LDFLAGS -lflexiblas" LAPACKLIB="$LDFLAGS -lflexiblas" '
```

Comment:
Here you can also use $LIBBLAS instead of -lflexiblas

Reply:
That does not seem to work for the `GCC-11.2.0` toolchain but with the other two. Thus, I have corrected only the `gfbf-2022a/b` ones. Please let me know if there is something I am missing in case of the `GCC-11.2.0` toolchain.